apps/sandbox/package.json

```diff
@@ -10,9 +10,10 @@
     "postinstall": "expo-yarn-workspaces postinstall"
   },
   "dependencies": {
-    "expo": "^33.0.0-alpha.web.2",
+    "expo": "32.0.0",
+    "expo-in-app-purchases": "1.0.0",
     "react": "16.8.3",
-    "react-native": "0.59.8"
+    "react-native": "^0.59.8"
```

Comment:
Since `sandbox` is a part of repository-wide Yarn workspace and we would always like to have one version of `react-native` inside that workspace I think it should be safe to use the strict one straight from `react-native-lab/react-native/package.json`

Reply:
Yeah I agree. Not sure why that ended up changing, I'll be sure to fix that.